src/multibanco/multibanco.js

```diff
@@ -0,0 +1,32 @@
+// 0. Get originKey
+getOriginKey().then(originKey => {
+    getPaymentMethods().then(paymentMethodsResponse => {
+        // 1. Create an instance of AdyenCheckout
+        const checkout = new AdyenCheckout({
+            originKey, // Mandatory. originKey from Costumer Area
+            environment: 'test',
+            amount: { currency: 'EUR', value: 1000 },
+            onAdditionalDetails: result => {
+                console.log(result);
+            },
+            onError: error => {
+                console.log(error);
+            },
+            onSubmit: (state, component) => {
+                makePayment(state.data).then(response => {
+                    // 3. present the voucher using the action object returned from /payments
+                    const voucher = new AdyenCheckout({
```

Comment:
We can reuse the same `AdyenCheckout` instance (`checkout`) to create new components from the payment actions, we can remove this.